test/specs/negative-login-test.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require("@wdio/globals");
+const LoginUtility = require("../utilities/login-utility");
+
+describe("Negative Login Test Cases", () => {
+  it("should show error messages for empty email and password fields", async () => {
+    await LoginUtility.loginUser("", "");
+    await LoginUtility.verifyRequiredFieldErrors();
+  });
+
+  it("should show error message for empty password", async () => {
+    await LoginUtility.loginUser("nadilovic@test.com", "");
```

Comment:
Credentials can be isolated in the constants in this or separate config data file to avoid hardcoding them directly into the test cases. This makes it easier to update these values in one place if needed.